packages/node/src/integrations/linkederrors.ts

```diff
@@ -0,0 +1,71 @@
+import { getCurrentHub } from '@sentry/hub';
```

Comment:
we always need `import { getCurrentHub } from '../hub';` in node, since the current hub could live on a domain.